section-engineering-education/engineering-education
# Batch Page views requests

Batch Page Views requests in order to reduce the number of times we fetch from the GA Page Views API and ultimately Google Data Analytics API.

**How to Test**:
1. Pull this branch down locally.
2. Run `hugo server -D` and navigate to `localhost:1313/` and view an authors profile page.
3. Navigate and pull down [this PR](https://github.com/section-io/ga-page-views-app/pull/11) locally as well. 
4. Run `node app.js`.
5. You should see page views displayed on each article.

**Alternatively** - Ping me in slack and I can walk through the demo so you don't have to pull two diff repos, run locally, etc. I will provide a demo.
